_posts/2018-07-25-Relay-Networking-Deep-Dive.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 layout: epic
-title: Relay Modern Network Deep Dive
+title: The Relay Network Deep Dive
```

Comment:
I've been wondering if we're far away enough that it doesn't need to be Relay Modern anymore, it's been over a year since it came out, and we've only ever used the terms relay classic internally when discussing the two